app/containers/Dashboard/index.js

```diff
@@ -24,14 +25,90 @@ const Dashboard = () => {
   const { loading, error, data } = useQuery(GET_ALL_STATUS, {
     variables: { customerId, statusDataTypes: ['CUSTOMER_DASHBOARD'] },
   });
+  const [toTime] = useState(
+    moment()
+      .valueOf()
+      .toString()
+  );
+  const [fromTime] = useState(
+    moment()
+      .subtract(24, 'days')
+      .valueOf()
+      .toString()
+  );
+
+  const {
+    loading: metricsLoading,
+    error: metricsError,
+    data: metricsData,
+    // refetch: metricsRefetch,
+  } = useQuery(FILTER_SYSTEM_EVENTS, {
+    variables: {
+      customerId,
+      fromTime,
+      toTime,
+      equipmentIds: [0],
+      dataTypes: ['StatusChangedEvent'],
+      limit: 100,
+    },
+  });
 
-  const titleList = ['Access Points', 'Client Devices', 'Usage Information'];
+  const pieChartTitle = ['Access Points', 'Client Devices', 'Usage Information'];
+  const lineChartTitle = [
+    'Inservice APs (24hours)',
+    'Client Devices (24hours)',
+    'Traffic (24hours)',
+  ];
 
-  if (loading) {
+  const [lineChartData] = useState({
+    inservicesAPs: { name: 'Inservice APs', value: [] },
+    clientDevices: {
+      is2dot4GHz: { name: '2.4GHz', value: [] },
+      is5GHzL: { name: '5GHz (L)', value: [] },
+      is5GHzU: { name: '5GHz (U)', value: [] },
```

Comment:
the radio series must be determined by the radios in `associatedClientsCountPerRadio` these radios are not guaranteed everytime and there can be `is5GHz`